views/index.pug

```diff
@@ -56,12 +56,12 @@ html
       | : Creates a new lead entry in the specified league from the data provided in the body.
     h1
       span.delete DELETE
-      | /leagues/:league/leads/{slug}
+      | /leagues/:leagueSlug/leads/{slug}
```

Comment:
This route doesn't work

![image](https://user-images.githubusercontent.com/5775716/83802227-5f194780-a678-11ea-90e1-8f8f2aa31331.png)